post_install.yml

```diff
@@ -43,3 +44,13 @@
     - name: Login to add token to kubeconfig
       shell:
         cmd: "oc --kubeconfig {{ kubeconfig_path }} login -u {{ credentials.json.username }} -p '{{ credentials.json.password }}'"
+
+    - name: Save credentials to file
+      ansible.builtin.copy:
+        content: "{{ credentials.json | to_yaml }}"
+        dest: "{{ dest_dir }}/{{ kubeadmin_vault_name }}"
+
+    - name: Save credentials to vault
+      # no_log: true
+      shell:
+        cmd: "ansible-vault encrypt --vault-password-file {{ kubeadmin_vault_password_file_path }} {{ dest_dir }}/{{ kubeadmin_vault_name }}"
```

Comment:
If we're using a FQCN for the `copy` task, I think it would make things more consistent if we added the `ansible.builtin.*` prefix to `shell` and all other relevant tasks too.

Reply:
Ah auto complete I'll remove the prefix on the copy :)